taca/illumina/Runs.py

```diff
@@ -74,12 +72,24 @@ def check_run_status(self):
             # Check if this job is done
             if os.path.exists(os.path.join(run_dir, demux_folder, 'Stats', 'DemultiplexingStats.xml')):
                 all_demux_done = all_demux_done and True
-                logger.info("Sub-Demultiplexing in {} completed.".format(demux_folder))
+                demux_log = os.path.join(run_dir, "demux_{}_bcl2fastq.err".format(demux_id))
+                errors, warnings, error_and_warning_messages = self._check_demux_log(demux_id, demux_log)
+                self.demux_summary[demux_id] = {'errors' : errors,
+                                                'warnings' : warnings,
+                                                'error_and_warning_messages' : error_and_warning_messages
+                                               }
+                if errors or warnings or error_and_warning_messages:
```

Comment:
```suggestion
                if errors or warnings:
```
would be better since `error_and_warning_messages` is not used in the following statement

Reply:
Fixed!